src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.js

```diff
@@ -12,22 +16,25 @@ const BaseAnchorForCommentsOnly = ({
     target,
     children,
     style,
+    translate,
     ...props
 }) => (
-    <Text
-        style={StyleSheet.flatten(style)}
-        accessibilityRole="link"
-        href={href}
-        hrefAttrs={{rel, target}}
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...props}
-    >
-        {children}
-    </Text>
+    <PressableWithContextMenu contextMenuItems={anchorContextMenuOptions(href, translate)}>
```

Comment:
Just Wrap this Anchor in `PressableWithSecondaryInteraction`
and Call the methods from `ReportActionContextMenu.js`.